### ISSUE TITLE
New seasons/episodes seem to not be detected

### ISSUE THREAD

**Issue by rafi-d** [pymedusa/Medusa#5397]
Hi,
For some reason with the last two re-newing shows I have - Simpsons, and Blindspot - Medisa failed to detect presence of new seasons. New episode were snatched, but the show-view was empty. 
Recovery from that situation was only after manual Force-Full-Update for the shows. Can this be fixed? 80% of my shows a not active, hoping to be auto-detected by Medusa when the time comes.

Attaching: before and after "force update" screenshots/
![medusa_neds_force_update_2018-10-13_133227](https://user-images.githubusercontent.com/2458280/46904474-54cb9800-ceed-11e8-9e31-1dbfdcb91068.png)

Medusa Info: | Branch: master Commit: 4614efc77151ded92ef458a09dec39f8bd5acfc6 Version: v0.2.10 Database: 44.12
-- | --
Python Version: | 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)]
SSL Version: | OpenSSL 1.0.2j 26 Sep 2016
OS: | Windows-10-10.0.17134
Locale: | en_US.cp1255
  |

**Comment by p0psicles** [comment]
I suspect the tvdb api is providing incomplete results regarding show updates. We depend on that, to calculate which seasons need to be refreshed

**Comment by medariox** [comment]
I have the same show and it updated correctly for me without any manual intervention.

**Comment by rafi-d** [comment]
Understood. If this is the case (a know issue with the APIs_, may you can overcome it with a bit of your own logic:
1. Snatch an episode
2. Check: is it  from a new season (sxe1 )?
3. If the current list/info is empty fir that season  - just auto-full-update it 

What do you think?

Thanks

Rafi

**Comment by a10kiloham** [comment]
This might be related to my issue? Manual refresh works fine but otherwise shows aren't pulling from theTVDB automatically anymore. For example, went to a show this AM and it was only showing 2 eps. Full refresh pulled a further 10 and had been out of sync for at least 3 weeks...

**Comment by a10kiloham** [comment]
https://github.com/pymedusa/Medusa/issues/5348

**Comment by a10kiloham** [comment]
It seems to be correctly logging a request, but yesterday it returned no new show. Today with manual refresh it instantly found a bunch of new eps that were on theTVDB yesterday already.

```
application.log.2:84776:2018-10-14 07:00:30 DEBUG    SHOWUPDATER :: [4614efc] Getting episodes for 295759
application.log.2:84777:2018-10-14 07:00:30 DEBUG    SHOWUPDATER :: [4614efc] Getting all episodes of 295759
application.log.2:84778:2018-10-14 07:00:30 DEBUG    SHOWUPDATER :: [4614efc] GET URL: https://api.thetvdb.com/series/295759/episodes/query?page=1 [Status: 200]
application.log.2:84993:2018-10-14 07:00:41 DEBUG    SHOWUPDATER :: [4614efc] 295759: Queueing show refresh for Supergirl
```

**Comment by rafi-d** [comment]
> it was only showing 2 eps. Full refresh pulled a further 10 and had been out of sync for at least 3 weeks...
^ sounds familiar ... :(

**Comment by ChangChiChun** [comment]
Same here but only one of my show's new seasons not showing(Money Heist Season 4)(Force-Full-Update won't fix it)
<img width="706" alt="截圖 2020-08-31 15 00 44" src="https://user-images.githubusercontent.com/10240691/91691772-cc1a7b80-eb9a-11ea-8c84-60f4d6ee9149.png">

**Comment by medariox** [comment]
TheTVDB has only 2 seasons listed for that show.

**Comment by p0psicles** [comment]
Might wanna try tvmaze for this show. They have 4 seasons and counting. They follow the Netflix order
